css/properties/text-decoration-thickness.json

```
@@ -6,18 +6,33 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-decoration-thickness",
           "spec_url": "https://drafts.csswg.org/css-text-decor-4/#text-decoration-width-property",
           "support": {
-            "chrome": {
-              "version_added": "87",
-              "notes": "The <code>text-decoration-thickness</code> property does not work unless either <code>text-underline-offset</code> is set to something other than <code>auto</code> or <code>text-decoration-color</code> is set to something other than <code>currentColor</code>. See <a href='https://crbug.com/1154537'>Chromium bug 1154537</a>"
-            },
-            "chrome_android": {
-              "version_added": "87",
-              "notes": "The <code>text-decoration-thickness</code> property does not work unless either <code>text-underline-offset</code> is set to something other than <code>auto</code> or <code>text-decoration-color</code> is set to something other than <code>currentColor</code>. See <a href='https://crbug.com/1154537'>Chromium bug 1154537</a>"
-            },
-            "edge": {
-              "version_added": "87",
-              "notes": "The <code>text-decoration-thickness</code> property does not work unless either <code>text-underline-offset</code> is set to something other than <code>auto</code> or <code>text-decoration-color</code> is set to something other than <code>currentColor</code>. See <a href='https://crbug.com/1154537'>Chromium bug 1154537</a>"
-            },
+            "chrome": [
+              {
+                "version_added": "87",
```

Since this note is not relevant beyond version 87, then let's not let this note be open-ended.

```suggestion
                "version_added": "87",
                "version_removed": "89",
```